easybuild/toolchains/gmpflf.py

```diff
@@ -45,3 +45,12 @@ class Gmpflf(Gmpich, OpenBLAS, FlexiBLAS,  ScaLAPACK, Fftw):
     """Compiler toolchain with GCC, MPICH, OpenBLAS, ScaLAPACK and FFTW."""
     NAME = 'gmpflf'
     SUBTOOLCHAIN = [Gmpich.NAME, Golf.NAME, Gfbf.NAME]
```

Comment:
@TopRichard The real problem is here, we shouldn't be using `golf` as a subtoolchain at all here.

We do this in `foss` for historical reasons, since for older versions of `foss` there was no FlexiBLAS yet, but here there's no need for it at all.

Just remove `Golf` from subtoolchains, and don't derive the `Gmpflf` class from `OpenBLAS` at all, and you shouldn't need the custom `__init__` at all, that should only be required in `foss` toolchain definition